1-org/envs/shared/README.md

```diff
@@ -29,7 +29,51 @@ You can choose not to enable the Data Access logs by setting variable `data_acce
 1. Merge changes to prod branch with `git checkout -b prod` and `git push origin prod`
     1. Review the apply output in your cloud build project https://console.cloud.google.com/cloud-build/builds?project=YOUR_CLOUD_BUILD_PROJECT_ID
 
-### Run terraform locally
+#### Setup to run via Jenkins
+1. Clone repo you created manually in Bootstrap: `git clone <YOUR_NEW_REPO-1-org>`
+1. Navigate into the repo `cd YOUR_NEW_REPO_CLONE-1-org` and change to a non prod branch `git checkout -b plan`
+1. Copy contents of foundation to new repo `cp -R ../terraform-example-foundation/1-org/* .` (modify accordingly based on your current directory).
+1. Copy the `Jenkinsfile` to the root of your new repository and replace the `_TF_SA_EMAIL` with the name of your Terraform Service Account in the `seed` project. (modify accordingly based on your current directory):
+   ```
+   cp ../terraform-example-foundation/build/Jenkinsfile .
+   sed -i 's/_TF_SA_EMAIL/TF_SERVICE_ACCOUNT_EMAIL/' Jenkinsfile
+   ```
+   **If using MacOS:**
+    ```
+    cp ../terraform-example-foundation/build/Jenkinsfile .
+    sed -i '.bak' 's/_TF_SA_EMAIL/TF_SERVICE_ACCOUNT_EMAIL/' Jenkinsfile
+    rm Jenkinsfile.bak
+    ```
+1. Copy the `tf-wrapper.sh` configuration file to the root of your new repository (modify accordingly based on your current directory):
+    ```
+    cp ../terraform-example-foundation/build/tf-wrapper.sh .
+    ```
+1. Ensure wrapper script can be executed `chmod 755 ./tf-wrapper.sh`.
+1. Rename `terraform.example.tfvars` to `terraform.tfvars` and update the file with values from your environment. (you can re-run `terraform output` in the 0-bootstrap directory to find these values). Make sure that `default_region` is set to a valid [BigQuery dataset region](https://cloud.google.com/bigquery/docs/locations).
+    ```
+    # Rename file
+    mv envs/shared/terraform.example.tfvars envs/shared/terraform.tfvars
+    # Edit the file to provide the necessary values
+    vi envs/shared/terraform.tfvars
+    ```
+1. Run this command to update the `backend.tf` file with the Terraform State GCS bucket name. Replace the `TF_STATE_GCS_BUCKET_NAME` with the name of your bucket (you can re-run `terraform output` in the 0-bootstrap directory to find these values).
```

Comment:
We shouldnt need this step now as Jenkinsfile takes care of this in setup stage?

Reply:
changed via f68b382